src/wallet.rs

```diff
@@ -215,6 +395,105 @@ mod tests {
         assert_eq!(from_keypair, to_keypair);
     }
 
+    #[test]
+    fn basic_from_builder() {
+        use std::fs;
+        let path: PathBuf = ".test-basic.key".into();
+        // Delete Any existing test wallet in case prev error
+        let _ = fs::remove_file(&path);
+
+        let password = String::from("password");
+        let tag = KeyTag::default();
+        let seed_words: Vec<String> = vec![
+            "drill".to_string(),
```

Comment:
There has to be a way where we can take a slice of `&str` instead of this repetition?

Reply:
It's just a test. Not too inclined to put any extra effort in using a const or something unless you think it is necessary?

Reply:
it just.. smells.. but ok